datastream-server/src/main/java/com/linkedin/datastream/server/zk/KeyBuilder.java

```diff
@@ -60,11 +60,15 @@ private KeyBuilder() {
    */
   private static final String DATASTREAM_TASK_CONFIG = CONNECTOR + "/%s/config";
 
+  /**
+   * lock node
+   */
+  protected static final String DATASTREAM_TASK_LOCK_ROOT_NAME = "lock";
```

Comment:
Did you mean to make it package private? That's less accessible than `protected` and would still allow you to access it in `ZkAdapter`.

Reply:
done